sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalogEventSuite.scala

```diff
@@ -209,4 +209,65 @@ class ExternalCatalogEventSuite extends SparkFunSuite {
     catalog.dropFunction("db5", "fn4")
     checkEvents(DropFunctionPreEvent("db5", "fn4") :: DropFunctionEvent("db5", "fn4") :: Nil)
   }
+
+  testWithCatalog("partition") { (catalog, checkEvents) =>
+    // CREATE
+    val path1 = Files.createTempDirectory("db_")
+    val uri1 = preparePath(path1)
+
+    val dbDefinition = createDbDefinition()
+
+    val storage = CatalogStorageFormat.empty.copy(
+      locationUri = Option(uri1))
```

Comment:
Is this table storage pointing db path? Maybe, do you want the following?
```scala
    val path1 = Files.createTempDirectory("db_")
    val path2 = Files.createTempDirectory(path1, "tbl_")
    val uri1 = preparePath(path1)
    val uri2 = preparePath(path2)

    // CREATE
    val dbDefinition = createDbDefinition(uri1)

    val storage = CatalogStorageFormat.empty.copy(
      locationUri = Option(uri2))
```

Reply:
yes, you are right